css/src/06sidebar.scss

```diff
@@ -214,7 +214,7 @@
 				}
 
 				&.high .section-title {
-					color: $red_overdue;
+					color: $color-error;
```

Comment:
I would not replace ```$red_overdue```. This color is matched to the red icons, and having the icons colored ```#ff0000``` looks weird.